doc/nrf/drivers/bme68x_iaq.rst

```diff
@@ -0,0 +1,39 @@
+.. _bme68x_iaq:
+
+BME68X IAQ driver
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use the BME68X IAQ driver to run the Bosch Sensor Environmental Cluster (BSEC) library in order to get Indoor Air Quality (IAQ) readings.
+
+.. note::
+   The BSEC library is distributed with a Bosch proprietary license (`BSEC license`_) that blocks it from been distributed as part of our nRF Connect SDK.
+
+To start using it you have to accept the license and enable the download with the following commands:
+
+.. code-block::
+
+   west config manifest.group-filter +bsec
+   west update
```

Comment:
The note could be removed in this case because line no: 15 is a continuation of line no:13.
```suggestion
The BSEC library is distributed with a Bosch proprietary license (`BSEC license`_) that prevents it from being part of |NCS|.
To start using it, you have to accept the license and enable the download with the following commands:

.. code-block::

   west config manifest.group-filter +bsec
   west update
```